### DISCUSSION TITLE
Axis Media Control 7_42 Error in image resolution and control positioning

### DISCUSSION THREAD

**Discussion by 3visionD** [AxisCommunications/discussions#135] [Axis Windows SDKs]
Hello there,
I am developing application that will read detected objects geo coordinates from PTZ camera, and I have notice “strange” display and positioning problem, what I believe is to be a bug. 
I have a form with width/height 1920x1080 and AMC control on this from. Origin at 0,0 and width/height 1920x1080. 
When HD stream from camera is displayed in AMC control vertical yellow bars are seen. This means that AMC control does not have width of 1920. Vertical resolution seams to be 1080 and that is OK. 
In my program I had to move AMC control origin 9 pixels to right and -13 pixels in upper direction to get symmetrical image looking from camera center. I have worked mostly with OBS and have not notice positioning problems. Can you look at this more carefully? 
Red lines in the image are drawn by AMC control so I can get symmetrical lines and compare there position against my screen.

private void crtaj_horizontalne_linije(int y)
        {
                     
            // Horizontalne linije
            amc.SetUserObject(1, 1, "0," + y.ToString() + ",9999," + y.ToString(), "test", 255, 255, -1, 0);
            amc.SetUserObject(2, 1, "0," +(9999- y).ToString() + ",9999," + (9999-y).ToString(), "test", 255, 255, -1, 0);
            //Ogiginal kod koji pokazuje grešku.
            //amc.SetUserObject(2, 1, "0,9994,9999,9994", "test", 255, 255, -1, 0);

            
        }

        private void crtaj_vertikalne_linije(int x)
        {
            // Vertikalne linije
            amc.SetUserObject(3, 1, x.ToString()+",0,"+ x.ToString()+",9990", "test", 255, 255, -1, 0);
            amc.SetUserObject(4, 1, (9999-x).ToString() + ",0," + (9999-x).ToString() + ",9990", "test", 255, 255, -1, 0);
            // amc.SetUserObject(4, 1, "9996,0,9996,9994", "test", 255, 255, -1, 0);
        }
![Axis Position and Origin problem](https://user-images.githubusercontent.com/83301268/205996870-abb4b8cd-736a-45e4-a946-1b8f9db7f84a.png)

**Comment by anders-axis** [comment]
Hi, is this still an issue? I could take a look.
